subsys/bluetooth/shell/audio.c

```diff
@@ -1579,6 +1609,9 @@ SHELL_STATIC_SUBCMD_SET_CREATE(audio_cmds,
 #endif /* CONFIG_BT_AUDIO_UNICAST */
 	SHELL_CMD_ARG(send, NULL, "Send to Audio Stream [data]",
 		      cmd_send, 1, 1),
+	SHELL_COND_CMD_ARG(CONFIG_BT_AUDIO_CAPABILITY, add_context, NULL,
+		      "<direction: sink, source> <context bitmask>",
+		      cmd_context, 3, 0),
```

Comment:
Please fix indendation :)

Reply:
Sorry, but I don't really see what's wrong with the indentation here. There are space characters in lines 1613 and 1614 to align it like previous command definitions.

Reply:
Should have been
```suggestion
	SHELL_COND_CMD_ARG(CONFIG_BT_AUDIO_CAPABILITY, add_context, NULL,
			   "<direction: sink, source> <context bitmask>",
			   cmd_context, 3, 0),
```

When the parameters are too long for a single line, they should be indented to the starting parenthesis

Reply:
Done